### ISSUE TITLE
Please don't replace .txt files

### ISSUE THREAD

**Issue by Beckfield** [tranxuanthang/lrcget#99]
I just heard about LRCGet and gave it a try.  It's a very impressive app.  The one problem I see, however, is that if I have existing untimed lyrics in a .txt file, LRCGet replaces it with the .lrc file without warning.

I tried setting the .txt file permissions to read-only, but that just prevents LRCGet from saving the lrc file at all.  This is not very intuitive behavior.

I would prefer that LRCGet save the .lrc file _alongside_ the existing .txt file.  Maybe there could be an option to choose this behavior.

**Comment by Beckfield** [comment]
> this is probably covered by #30

I don't think that's really the same thing.  My issue is that existing lyric.txt files are deleted when LRCGet downloads .lrc files.  That's very bad behavior, especially when no notice is given.

**Comment by ClaireCJS** [comment]
I agree! I want TXT *and* LRC, for different reasons. Deleting files is very bad. Is this happening right now?

**Comment by ClaireCJS** [comment]
Plus also, I hand-edit my TXT and LRC.

Sometimes i put work into it.

are the now being replaced with a downloaded version, if i run this successively?

because i want to run this on my whole collection every 6 months or so

it can skip downloading lrc for files that already have an lrc
it can skip downloading txt for files that already have an txt

**Comment by tranxuanthang** [comment]
I agree about this, it should not automatically delete or replace LRC or TXT files without asking, and an option for skipping downloading lyrics for files that already have an txt would be nice. I'm in my busy phase but hopefully the change will be there in the next update somewhere in the next month :-)
